CONTRIBUTING.md

```diff
@@ -95,30 +98,91 @@ we would appreciate if your pull request applies to the following points:
-      well as commits have been signed-off and
+    * _Intellectual Property Validation_ verifying the [Eclipse CLA](#eclipse-contributor-agreement) 
+      has been signed as well as commits have been signed-off and
     * _Continuous Integration_ performing various test conventions.
 
+## Project and Milestone Planning
+
+Milestones are planned for about 6 to 8 weeks. Future milestones describe an outlook on features to 
+be considered. The transparent process on GitHub enables participation. To decide what features and
+issues will be targeted, user requirements and sponsors are considered. Thus, everyone is able to
+influence the addressed developments and its priorities! 
+
+### Projects
+
+The [Github Projects page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/projects)
+provides a general overview of the project's working items. Every new issue needs to be assigned to
+the ["Dataspace Connector" project](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/projects/1).
+There, an issue passes five stages:
+* `Open`: All assigned issues will automatically have this status.
+* `Backlog`: An open issue does not automatically imply that it is going to be addressed in upcoming
+  milestones and releases - maybe someone opened it to report a bug or ask a question. An issue that 
+  will be resolved must be put to this column.
+* `In progress`: Reopened issues and pull requests or new pull requests will be added to this column
+  automatically if they previously have been assigned to the project.
+* `Review in progress`: This covers pull requests that are no longer marked as draft and have 
+  pending approvals.
+* `Done`: All resolved or closed issues and pull requests will remain here.
+
+If an issue covers a topic or the response to a question that may be interesting for other 
+developers or contributors, it should be converted to a discussion and not be closed!
+
+### Milestones
+
+Milestones are organized at the 
+[Github Milestones page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/milestones).
+They are numbered in ascending order, e.g. `Milestone 1`, `Milestone 2`, ... Closed milestones can
+always be viewed in the according 
+[tab](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/milestones?state=closed).
+
+The ongoing milestone has a due date - that is **no guarantee** that all linked issues will be 
+resolved by then. 
+Further milestones, that are already known, can be created any time, and issues can be assigned
+to them. However, for every new milestone, a due date will be voted in the committers round only as
+soon as the current milestone is closed.
+
+The milestone `Milestone Scoping` is a permanent milestone without a due date. Issues that should be 
+considered for future milestones but already need to be targeted for the current one will be 
+assigned here. This way, contributors, users, and adopters can track the progress.
```

Comment:
not sure if I get that right...
is it more like "Milestone Scoping = accepted issue, but not mapped to fix Milestone so far"
or more like "Milestone Scoping = things we need to refine during the current milestone"
or ...?

Reply:
If I understood @mspiekermann right, this label "scoping" should not cover the complete backlog but issues in the backlog that should be adressed already during the milestone, but should actually be resolved in some further milestone - as some kind of preparing. I personally don't think that this should actually be a category of issues because either you work on something or you don't - but I am open to add this to the milestone planning. I don't know if this is a proper way without creating confusion? However, I argue to not use labels for this. Labels should be for categorization, not milestone planning.

Reply:
Do you understand the meaning? Do you also consider this as important, and if yes, is there a better naming? `Assessment queue`, `Pending evaluation`, `Multi-milestone scoping`, etc.? Or is such an issue just added to the project's `in progress` column and then everyone knows that this is been worked on?